ci/build.ps1

```diff
@@ -167,14 +167,15 @@ if (-not $NoTests) {
   # Old tests
   # Add MSYS to path, required for e.g. `find` used in test scripts.
   # But would break functionaltests, where its `more` would be used then.
-  $OldPath = $env:PATH
-  $env:PATH = "C:\msys64\usr\bin;$env:PATH"
-  & "C:\msys64\mingw$bits\bin\mingw32-make.exe" -C $(Convert-Path ..\src\nvim\testdir) VERBOSE=1 ; exitIfFailed
-  $env:PATH = $OldPath
-
-  if ($uploadToCodecov) {
-    bash -l /c/projects/neovim/ci/common/submit_coverage.sh oldtest
-  }
+  
+  # $OldPath = $env:PATH
```

Comment:
If we're going to leave the code in, could you add a comment explaining why it's commented out so that new devs (like me) understand the logic a little better? Alternatively, what's stopping us from removing it altogether?

Reply:
Absolutely, good call. I don't think we want to remove the code since the end goal is to actually use it, this was just intended as a temporary fix until the oldtests are functional again on windows (they currently hang on some test making all windows jobs time out). But maybe it's better to just try to fix the test directly instead :thinking: . I'll need to think about it. Drafting this for now.